Python/sysmodule.c

```diff
@@ -2861,6 +2861,62 @@ _PySys_InitCore(PyThreadState *tstate, PyObject *sysdict)
     return _PyStatus_ERR("can't initialize sys module");
 }
 
+#ifdef HAVE_FORK
+extern int _PyIO_buffered_at_fork_reinit(PyObject *);
+
+static int
+stdio_at_fork_reinit(_Py_Identifier *key)
+{
+
+    int ret = 0;
+
+    PyObject *isatty = NULL;
+    PyObject *buffer = NULL;
+
+    PyObject *stdio = _PySys_GetObjectId(key);
+
```

Comment:
stdio can be NULL or Py_None. You should do nothing in this case.

Reply:
Thanks, done.